src/data/ast.test.ts

```diff
@@ -15,5 +15,30 @@ describe('ast', () => {
       // this is formatted like this to match how pgsql generates its sql
       expect(toSql.statement(stm)).toEqual('SELECT (count (*) )  FROM foo');
     });
+
+    // https://github.com/grafana/clickhouse-datasource/issues/714
+    it('does not error when brackets/macros/variables are present', () => {
```

Comment:
Let's add more tests in a follow-up. I'd add more built-in ClickHouse functions, [query params](https://clickhouse.com/docs/en/interfaces/cli#cli-queries-with-parameters)